src/index.js

```diff
@@ -0,0 +1,6 @@
+import './style.css';
+import './dom.js';
+
+const dom = document.getElementById('dom');
+
+dom.innerHTML = '<li>webpack setup</li>';
```

Comment:
- [ ] There is no HTML element with id 'dom'. As a result, the app throws a browser console error. Kindly fix this. You can use this file to create elements dynamically :pray: 

![Screenshot from 2021-11-17 00-37-31](https://user-images.githubusercontent.com/64894927/142070651-dad8467f-3106-41f2-9d0d-93932fb3116e.png)